src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -112,11 +112,7 @@ export default [
                     if (!Clipboard.canSetHtml()) {
                         Clipboard.setString(parser.htmlToMarkdown(content));
                     } else {
-                        // Thanks to how browsers work, when text is highlighted and CTRL+c is pressed, browsers end up doubling the amount of newlines. Since the code in this file is
-                        // triggered from a context menu and not CTRL+c, the newlines need to be doubled so that the content that goes into the clipboard is consistent with CTRL+c behavior.
-                        // The extra newlines are stripped when the contents are pasted into the compose input, but if the contents are pasted outside of the comment composer, it will
-                        // contain extra newlines and that's OK because it is consistent with CTRL+c behavior.
```

Comment:
Did we ever get to the bottom of why this comment was added or whether it's accurate? Maybe this wasn't browser behavior after all?

Reply:
It's very possible I was misunderstanding the behavior when I wrote that comment :D